server/cli/src/backup/backup.rs

```diff
@@ -137,3 +149,31 @@ fn copy_sqlite_files(
 
     Ok(())
 }
+
+fn cleanup_backups(
+    backups_dir: &PathBuf,
+    max_number_of_backups: Option<u32>,
+) -> Result<(), BackupError> {
+    let Some(max_number_of_backups) = &max_number_of_backups else {
+        return Ok(());
+    };
+
+    let mut paths: Vec<PathBuf> = fs::read_dir(backups_dir)?
+        .into_iter()
+        .filter_map(Result::ok)
+        .map(|e| e.path())
+        .filter(|f| f.is_dir())
+        .collect();
+
+    paths.sort();
+
+    for path in paths
+        .iter()
+        .take(paths.len() - *max_number_of_backups as usize)
```

Comment:
Getting this error until I have more than max number of backups... probably should check for paths.len() >= max_backups first 😁 
```
thread 'main' panicked at cli/src/backup/backup.rs:172:15:
attempt to subtract with overflow
```

Reply:
Ohh no, i forgot to test this use case: 36b8266e48b95638f9556558b93a68e18fa453d6

Reply:
thank you